src/components/20-molecules/footer-small/index.js

```diff
@@ -43,45 +44,34 @@ class AXAFooterSmall extends InlineStyles {
     if (this.dynamic) {
       ev.preventDefault();
       this.onLanguageClick(languageIndex);
-      this.shadowRoot
```

Comment:
I guess the removal of this block is the reason, why the active-language indicator does not update anymore.

To be seen here: https://axa-ch.github.io/plib-feature/bugfix/footer-small-dynamic-children/?path=/story/molecules-footer-small-react--footer-with-callbacks-on-language

And here: https://axa-ch.github.io/plib-feature/bugfix/footer-small-dynamic-children/?path=/story/molecules-footer-small-demos--footer-small-dynamic-links

I checked with the current prod and there it works. Interestingly, this dynamic example from your branch seems to work: 
https://axa-ch.github.io/plib-feature/bugfix/footer-small-dynamic-children/?path=/story/molecules-footer-small-react--footer-with-dynamic-change-of-children-upon-language-change

Reply:
Fixed with new commit, thanks!

Reply:
Again fixed with new commit, sorry...